overlord/ifacestate/ifacemgr.go

```diff
@@ -62,6 +62,7 @@ func Manager(s *state.State, extra []interfaces.Interface) (*InterfaceManager, e
 	runner.AddHandler("setup-profiles", m.doSetupProfiles, m.doRemoveProfiles)
 	runner.AddHandler("remove-profiles", m.doRemoveProfiles, m.doSetupProfiles)
 	runner.AddHandler("discard-conns", m.doDiscardConns, m.undoDiscardConns)
+	runner.AddHandler("discard-namespace", m.doDiscardNamespace, nil)
```

Comment:
Reading these changes it's unclear to me whether this command is about mount namespaces specifically, or other namespaces too. What's the case here? The task and the tool need to be named after their actual intended functionality, and it must be realistic going forward.